scm/doc/TechGuide/chap_quick.rst

```diff
@@ -453,16 +455,34 @@ namelists in ``../etc/case_config``. A default physics suite is provided as a us
 variable in the script and default namelists and tracer configurations
 are associated with each physics suite (through ``../src/suite_info.py``), so, technically, one
 must only specify a case to run with the SCM when running just one
-integration. For running multiple integrations at once, one need only
-specify one argument (``-m``) which runs through all permutations of supported
-suites from ``../src/suite_info.py`` and cases from ``../src/supported_cases.py``. The run script’s options are described
-below where option abbreviations are included in brackets.
+integration. For example, to run the "BOMEX" case:
+
+.. code:: bash
+
+  ./run_scm.py -c bomex
+
+For running multiple integrations at once, the run script can accept a file that contains a list of tests to run.
+The file ``ccpp-scm/test/rt_test_cases.py`` contains the full list of regression test cases, so you could run that list
+of tests with the following command:
+
+.. code:: bash
+
+ ./run_scm.py -f ../../test/rt_test_cases.py
```

Comment:
Something we need to discuss are the SCM RTs.
[rt_test_cases.py](https://github.com/NCAR/ccpp-scm/blob/main/test/rt_test_cases.py) needs to be extended to 
- include the v7 tests, which we need to decide on. Same SDF/case combos as in v6 plus some new cases? 
(*Note* we have a separate CI test that use[ the DEPHY repository](https://github.com/GdR-DEPHY/DEPHY-SCM), but there aren't any comparisons in that test, it just ensures the SCM works with the DEPHY repo. When I set this up I didn't do a deep evaluation on the case from the DEPHY repo, just more of a technical test that teh SCM could use the latest and greatest DEPHY cases. @bluefinweiwei has used the SCM with cases in the DEPHY repo for T&E, so I think it makes more sense to use a case that she is confident in. And then add a comparison step to the DEPHY CI test.
(**Note** We have a separate CI test for the UFS case generation. This test just creates the UFS case files from staged UFS RT data. It doesn't run the SCM with that case. We need to either expand this test to run the SCM and compare to baselines, OR leave this CI test as is and add a staged UFS case to the SCM, to be included in rt_test_cases.py?)
- move the "supported v6" into the "unsupported" section. As long as these still work, we should include them in our testing suite.
- Beef up [the comparison step in the CI RT script](https://github.com/NCAR/ccpp-scm/blob/main/.github/workflows/ci_run_scm_rts.yml#L197) to create plots of fields that differ, and upload these as a github artifact.
- We only run the CI tests with GNU. Intel compilers aren't available on the guthub servers we do the CI, but we could create a Intel Docker container. TBH, for the RT CI test, we don't need to build the software stack as part of the test since [ci_build_scm_ubuntu_22.04.yaml](https://github.com/NCAR/ccpp-scm/blob/main/.github/workflows/ci_build_scm_ubuntu_22.04.yml) already does this, we just need to run the SCM and compare to baselines.